handlers/main.yaml

```diff
@@ -0,0 +1,6 @@
+---
+- name: reload systemd
+  systemd:
+    daemon_reload: yes
+    name: efi-clone.service
+    state: reloaded
```

Comment:
I don't think this does anything with the given service file, does it?

Reply:
Yes, you are right. I reread the systemd docs and enabling a service also does a reload, so this is not necessary.

Reply:
That is not what I meant.
I thought the `daemon_reload` was necessary, but the `state: reloaded` not (as that tells the service to reload its config file or such, but our service does not implement this).
Can you provide a link to the docs that explain that `daemon_reload` is not required?

Reply:
From `man systemctl` in the section for `enable:`

> After the symlinks have been
           created, the system manager configuration is reloaded (in a
           way equivalent to daemon-reload), in order to ensure the
           changes are taken into account immediately.